app/src/components/widgets/singleLevelWidgets/charts/failedCasesTrendChart/config/getConfig.js

```diff
@@ -40,6 +40,7 @@ export const getConfig = ({ content, isPreview, formatMessage, positionCallback,
       colors: {
         failed: COLOR_FAILED,
       },
+      launchesId: itemsData.map((item) => item.id),
```

Comment:
`launchIds`

Reply:
I think it is not the best idea to hold it here. It will be recalculated every time when widget will regenerate config.
I suggest to map `id`s directly in `onChartClick` method.